test/boost/row_cache_test.cc

```diff
@@ -4219,3 +4219,58 @@ SEASTAR_TEST_CASE(test_reading_of_nonfull_keys) {
                     .produces(m1);
         });
 }
+
+SEASTAR_TEST_CASE(test_populating_cache_with_expired_and_nonexpired_tombstones) {
+    return do_with_cql_env_thread([](cql_test_env& env) {
+        replica::database& db = env.local_db();
+        auto& mm = env.migration_manager().local();
+        tests::reader_concurrency_semaphore_wrapper semaphore;
+
+        sstring ks_name = "ks";
+        sstring table_name = "table_name";
+
+        schema_ptr s = schema_builder(ks_name, table_name)
+                .with_column("pk", int32_type, column_kind::partition_key)
+                .with_column("ck", int32_type, column_kind::clustering_key)
+                .build();
+        mm.announce(
+            mm.prepare_new_column_family_announcement(s, api::new_timestamp()).get(),
+            mm.start_group0_operation().get()
+        ).get();
```

Comment:
Simpler:
```
env.execute_cql("CREATE TABLE ...").get();
env.require_table_exists(...).get;
```

Reply:
(also safer, best not to rely on mm internals in tests)

Reply:
Thanks, fixed